public/src/utils.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
 
+var winston;
```

Comment:
Pass winston/console in like is done with xregexp

Reply:
Right now winston will be defined as a global variable in the browser. Do you see how XRegExp is passed into the factory function. On node it's required, in browser it's passed in from window.XRegExp

So add a `console` or `logger` parameter to the factory function, and pass winston of console in respectively, like XRegExp is handled